bluebrain/repo-bluebrain/packages/neurodamus/package.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from llnl.util import tty
+
+from spack.package import *
+
+from .neurodamus_model import NeurodamusModel, copy_all, make_link
+
+
+class Neurodamus(NeurodamusModel):
+    """The next-generation AllInOne Neurodamus deployment.
+
+    Neurodamus includes BBP's simulation suite with all internal sim models.
+    """
+
+    homepage = "https://bbpgitlab.epfl.ch/hpc/sim/neurodamus-models"
+    git = "ssh://git@bbpgitlab.epfl.ch/hpc/sim/neurodamus-models.git"
+
+    version("develop", submodules=True)
+    # Let the version scheme be different to avoid users loading it by mistake and getting puzzled
+    version("2023.04", branch="bump/2023-april", submodules=True)
```

Comment:
No branches in versions, please. What do you mean by
> Let the version scheme be different
?

Is that "prefer calendar versions to distinguish against previous `neurodamus-*`?